src/lime/extension/DisplayArgsProvider.hx

```diff
@@ -21,9 +21,9 @@ class DisplayArgsProvider
 	{
 		updateArgumentsCallback = provideArguments;
 
-		if (arguments != null)
+		if (parsedArguments != null)
```

Comment:
This didn't work on second activation because of the `this.arguments != arguments` check in `update()`.